src/cmdlib.sh

```diff
@@ -6,15 +6,18 @@ fatal() {
 
 preflight() {
     # Verify we have all dependencies
-    local deps=$(grep -v '^#' /usr/libexec/coreos-assembler/deps.txt)
-    if ! rpm -q ${deps} >/dev/null; then
+    local deps=$(grep -v '^#' /usr/lib/coreos-assembler/deps.txt)
+    # Explicitly check the packages in one rpm -q to avoid
+    # overhead, only drop down to individual calls if that fails.
+    # We use --whatprovides so we handle file paths too.
+    if ! rpm -q --whatprovides ${deps} &>/dev/null; then
```

Comment:
Minor: there's a `--quiet` option at least, which quelches both the happy and ENOENT case.